server/index.js

```diff
@@ -7,11 +7,12 @@ const app = express();
 
 app.use(bodyParser.urlencoded({extended: true}));
 app.use(bodyParser.json());
-//REMEMBER TO ADD STATIC PATH
+app.use(express.static(__dirname + '/../public'));
 
 //REMEMBER TO FIX FIND ONE BELOW!!!!!!!!!!!!
-app.get('/videoData', (req, res) => {
-  mongoModel.GameModel.findOne({id: 1})
+app.get('/api/mediaData/:id', (req, res) => {
+  let id = req.params.id
+  mongoModel.GameModel.findOne({id})
     .then((data) => {
       res.json(data);
     })
```

Comment:
Is there something better we can do when an error occurs besides logging it to the console? As a user, I want to know why my get request to id 90000001 didn't work.